pkg/operator/podidentity/podidentitywebhook_controller.go

```diff
@@ -209,7 +233,7 @@ func Add(mgr, rootCredentialManager manager.Manager, kubeconfig string) error {
 		if !ok {
 			return fmt.Errorf("failed to convert runtime.Object to client.Object")
 		}
-		informer, err := cache.GetInformer(context.TODO(), co)
+		informer, err := controllerCache.GetInformer(context.TODO(), co)
```

Comment:
Thread through a context instead of using `context.TODO()` - this might be pre-existing code but if everyone pitches in to help refactor this we can squash the tech debt faster.